pkg/apiserver/compression.go

```diff
@@ -3,18 +3,22 @@ package apiserver
 import (
 	compressGzip "compress/gzip"
 	"fmt"
-	ginGzip "github.com/gin-contrib/gzip"
-	"github.com/gin-gonic/gin"
 	"io"
 	"strconv"
+
+	ginGzip "github.com/gin-contrib/gzip"
+	"github.com/gin-gonic/gin"
 )
 
+// CompressionSettings allow the enabling of gzip support for requests and responses. By default compressed requests are accepted, and compressed responses are returned (if suitable).
 type CompressionSettings struct {
-	Level         string                     `cfg:"level" default:"default" validate:"oneof=none default best fast 0 1 2 3 4 5 6 7 8 9"`
-	Decompression bool                       `cfg:"decompression" default:"true"`
-	Exclude       CompressionExcludeSettings `cfg:"exclude"`
+	Level         string `cfg:"level" default:"default" validate:"oneof=none default best fast 0 1 2 3 4 5 6 7 8 9"`
+	Decompression bool   `cfg:"decompression" default:"true"`
+	// Exclude some paths from compression for more advanced use cases (if you already are serving compressed/uncompressed data on those paths)
```

Comment:
Why are you documenting the most unimportant field, but leave out stuff from the documentation like:
- what does decompression mean? What does that imply?
- if I don't want to use the default level, what is the range of valid values?

Also I find saying "more advanced use cases" not very nice... So our support for the user stops literally at the point they need it the most. Why not simply say something like "Exclude files by path, extension, or regular expression from being considered for compression. Useful if you are serving a format unknown to Gosoline which doesn't compress well from a path for example."